test/exceptions/Makefile

```diff
@@ -94,6 +94,10 @@ $(ROOT)/refcounted.done: $(ROOT)/refcounted
 	$(QUIET) $<
 	@touch $@
 
+ifeq (ldmd,$(findstring ldmd,$(HOST_DC)))
+# LDC: Make sure allocation intended to provoke exception is not elided.
+$(ROOT)/invalid_memory_operation: DFLAGS+=-disable-gc2stack
+endif
```

Comment:
I don't think this should be here upstream, way too LDC-specific.

Reply:
What is the problem with this ?